src/crypto/StreamingAES.ts

```diff
@@ -121,6 +122,7 @@ export class StreamingDecryption {
             if (!chunk.length) {
                 return callback();
             }
+            //TODO: THIS DOESN'T WORK WITH VERSION 2 DATA YET
```

Comment:
I didn't really think it was worth all the effort to implement it now, since we don't let people streaming decrypt. It isn't a trivial amount of work because I have to keep reading and checking that I got enough data from the stream to be able to fully parse the header. Obviously doable, but since this code can't be called, I figured there wasn't any harm in waiting until it was supported (at which point we probably have version 3) to do it all.